tests/marray.test.ts

```diff
@@ -0,0 +1,35 @@
+import * as fc from 'fast-check';
```

Comment:
Hmm. Why place the file in `tests` not next to the source like the other specs?

Reply:
I saw that the tests for the tree were there so I put it there, moved it to `src` for `MArray` consistency.

Commit: [98a1271](https://github.com/hfour/yads/pull/33/commits/98a127190020004645a81f370e7473c081427f41)